### DISCUSSION TITLE
请问client端有支持webdav的计划吗？

### DISCUSSION THREAD

**Discussion by ghost** [juicedata/juicefs#1378] [Q&A] [answered]
请问client端有支持webdav的计划吗？ 如果没有，我这边想试试做二次开发，稳定运行的话提个PR

**Comment by xiaogaozi** [comment, marked as answer]
JuiceFS 已经支持 WebDAV，请参考文档：https://juicefs.com/docs/zh/community/how_to_setup_object_storage#webdav

**Reply by davies** [reply]
楼主说的是 通过 WebDAV 来访问 JuiceFS里面的数据，建了一个 issue： https://github.com/juicedata/juicefs/issues/1441

JuiceFS 的商业版已经支持它，稍后我们会移植到社区版。
